tests/acceptance/02_classes/02_functions/isnewerthantime.cf

```diff
@@ -0,0 +1,64 @@
+body common control
+{
+  inputs => { "../../default.cf.sub" };
+  bundlesequence => { default("$(this.promise_filename)") };
+  version => "1.0";
+}
+#######################################################
+
+bundle agent check
+{
+  vars:
+    "test_file"
+      string => "$(this.promise_dirname)/isnewerthantime_file.txt";
```

Comment:
You can consider creating this file in an `init` bundle and setting the `mtime`, all using the `files` promise.

Reply:
Implementation is using the derived mtime via filestat()